ansible/roles/proxy/tasks/download.yml

```diff
@@ -32,3 +32,12 @@
     - { urlp: "{{ apache_mirror.stdout }}/hadoop/common/hadoop-{{ hadoop_version }}", fn: "{{ hadoop_tarball }}", sum: "{{ hadoop_sha256 }}" }
     - { urlp: "{{ apache_mirror.stdout }}/maven/maven-3/{{ maven_version }}/binaries", fn: "{{ maven_tarball }}", sum: "{{ maven_sha256 }}" }
     - { urlp: "https://github.com/github/hub/releases/download/v{{ hub_version }}", fn: "{{ hub_tarball }}", sum: "{{ hub_sha256 }}" }
+
+- name: "Download javax.activation-api for Hadoop 3"
+  maven_artifact:
```

Comment:
Neat, didn't know ansible had this feature. Would be nice to add a comment saying this is currently needed to run hadoop with Java 11.

Reply:
Done!